assets/base/i18n/en.json

```diff
@@ -158,16 +158,21 @@
   "mobile.channel_drawer.search": "Jump to...",
   "mobile.channel_info.alertMessageDeleteChannel": "Are you sure you want to archive the {term} {name}?",
   "mobile.channel_info.alertMessageLeaveChannel": "Are you sure you want to leave the {term} {name}?",
+  "mobile.channel_info.alertMessageConvertChannel": "When you convert {displayName} to a private channel, history and membership are preserved.\n\nPublicly shared files remain accessible to anyone with the link. Membership in a private channel is by invitation only.\n\nThe change is permanent and cannot be undone.",
   "mobile.channel_info.alertNo": "No",
+  "mobile.channel_info.alertCancel": "Cancel",
+  "mobile.channel_info.alertOk": "Ok",
```

Comment:
I think you are right with your previous comment, so perhaps we can re-use an existing translation for `Ok` and `Cancel`

Reply:
Got it, will remove these and use an existing translation instead!